server/system/types/user.go

```diff
@@ -37,8 +37,16 @@ type (
 	}
 
 	UserMeta struct {
-		// @todo remove, obsolete.
-		Avatar string `json:"avatar,omitempty"`
+		// User's profile avatar photo attachment ID
+		AvatarID uint64 `json:"avatarID,string"`
+
+		// User's profile avatar initials attachment ID
+		AvatarInitialsID uint64 `json:"AvatarInitialsID,string"`
+		// User's profile avatar custom initials
+		AvatarInitials string `json:"avatarInitials,omitempty"`
+		// User's profile Initial background and text color
+		AvatarInitialsTextColor string `json:"avatarInitialsTextColor,omitempty"`
+		AvatarInitialsBgColor   string `json:"avatarInitialsBgColor,omitempty"`
```

Comment:
Do we need to remember all of this?
I wouldn't save these. If you really want this, you can keep it or you can utilize .png metadata